.gitignore

```diff
@@ -16,3 +16,5 @@ pom.xml.asc
 .lein-*
 .cpcache
 .nrepl-port
+/.idea
```

Comment:
This is a convenience for IntelliJ/Cursive users that prevents project files from being committed.